testsuite/features/support/commonlib.rb

```diff
@@ -68,80 +68,21 @@ def compute_image_name
 # This function computes a list of reposyncs to avoid killing, because they might be involved in bootstrapping.
 #
 # This is a safety net only, the best thing to do is to not start the reposync at all.
-# rubocop:disable Metrics/MethodLength
-# rubocop:disable Metrics/BlockLength
 def compute_list_to_leave_running
   do_not_kill = []
-  if $long_tests_enabled
```

Comment:
No. The products for auto-installation tests should not disappear.

We need to guarantee that these products are added to the whitelist when we are testing auto-installation and that they are not when we don't.
￼
It is exactly same situation as for the service pack migration

The argument "we are going to run the service pack migration by default" is invalid, the test `  if $service_pack_migration_enabled` tells exactly the contrary.

Don't worry about putting duplicate packages in the list, the list ist reduced at the end anyway thanks to `.uniq`.

Reply:
I don't understand your message :)
I think nothing has been disappeared just well refactored :)

Reply:
OK, I see Eric's point; they are needed only at HEAD because we run such a test only at that branch.
I have just pushed a commit to reenable them into `do_not_kill`